.github/workflows/deploy_cms.yaml

```diff
@@ -0,0 +1,74 @@
+name: Deploy CMS
+
+on:
+  push:
+        branches: 'main'
+        # Run only if there are at least one change matching the following paths
+        paths:
+          - 'apps/strapi-cms/*'
+          - '.github/workflows/deploy_cms.yaml'
+
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+        type: choice
+        options:
+        - info
+        - warning
+        - debug
+      environment:
+        description: 'The environment used as target'
+        type: choice
+        required: true
+        default: prod
+        options:
+        - prod
+
+defaults:
+  run:
+    shell: bash
+    working-directory: apps/strapi-cms
+
+permissions:
+  id-token: write   # This is required for requesting the JWT
+  contents: read    # This is required for actions/checkout
+
+jobs:
+  deploy:
+    name: Docker CMS
+    runs-on: ubuntu-latest
+    continue-on-error: false
+    environment: ${{ inputs.environment }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+    
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@68827325e0b33c7199eb31dd4e31fbe9023e06e3 # v3.0.0
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@f95db51fddba0c2d1ec667646a06c2ce06100226 # v3.0.0
```

Comment:
Instead of making on your own these steps (and some of the following Docker related steps), you could use the [build-push-action](https://github.com/docker/build-push-action). What do you think?

Reply:
yes, I had seen this suggested action. It does not save us these steps (docker/setup-qemu-action, setup-buildx-action, Login to ECR) but it includes the build, tag and push steps in a single action. 
Example:
  - name: Build Push and Tag
    uses: docker/build-push-action@v4
    with:
      context: .
      file: ./Dockerfile
      push: true
      tags: ${{ steps.login-ecr.outputs.registry }}/cms-strapi:latest
      provenance: false

what do you think about it ?

Reply:
I found [an old project we worked on](https://github.com/pagopa/pn-local-emulator/blob/1771071b3274203eec83636c55733984311158c9/.github/workflows/main.yaml#L54-L57), where we have a similar use case: we need to create and push a Docker image and there we used only the build-push-action I mentioned.

I'm sure that something could be evolved, indeed we used `v3` of that action while now the major version is `v5`. But, could you investigate a little to see if there is possible to omit those steps that are using setup-qemu-action and setup-buildx-action?

Reply:
change present in the latest commit